src/benchcab/benchcab.py

```diff
@@ -347,8 +360,7 @@ def fluxsite_run_tasks(self, config_path: str):
         else:
             fluxsite.run_tasks(tasks)
 
-        tasks_failed = [task for task in tasks if not task.is_done()]
-        n_failed, n_success = len(tasks_failed), len(tasks) - len(tasks_failed)
+        n_tasks, n_success, n_failed, all_complete = task_summary(tasks)
```

Comment:
`task_summary` was a helper function developed for this, before we started using hpcpy to handle the downstream submission. Still useful.